### ISSUE TITLE
Improve model cards for pretrained weights

### ISSUE THREAD

**Issue by calebrob6** [microsoft/torchgeo#1042]
With the new v0.4 update to TorchGeo we have the ability to easily add pre-trained model weights (e.g. https://github.com/microsoft/torchgeo/blob/main/torchgeo/models/resnet.py#L28). Currently, each set of weights comes with an [`AugmentationSequential`](https://github.com/microsoft/torchgeo/blob/811a12c105ad839e05343ee243804b3694d36816/torchgeo/transforms/transforms.py#L19) instance that describes the preprocessing that the model expects, but don't include the proper normalization values (or a description of what range of values the model expects as input). These need to be retroactively added for all existing weights, and, ideally, verified (by trying to match reported downstream task performance with the model weights).

For example, the Sentinel 2 models from the SSL4EO papers likely expect uint12 Sentinel 2 values that are divided by 10,000 -- https://github.com/zhu-xlab/SSL4EO-S12/blob/main/src/benchmark/transfer_classification/datasets/EuroSat/eurosat_dataset.py#L97. This step should be added to the  [current set of transforms](https://github.com/microsoft/torchgeo/blob/main/torchgeo/models/resnet.py#L18) and tested.

If verification of the model weights is done on downstream tasks, we should also report those performance numbers vs. relying on the numbers reported in the associated papers.

**Comment by clkruse** [comment]
I'd find this to be quite useful too! Found this issue since I was digging through code to figure out what normalization strategy to use for the SSL4EO pretrained models.

**Comment by calebrob6** [comment]
It seems like this is a popular idea, see https://github.com/zhu-xlab/SSL4EO-S12/issues/2 ;)
